README.md

```diff
@@ -33,7 +33,7 @@ A new about page for Aragon DAOs
    npm run start:ipfs:template -- --client false
    ```
 
-4. Start yet another terminal to run the customized client supporting Home setting
+4. Start yet another terminal to run the customized client supporting About setting
```

Comment:
I agree, the README.md will need to be reviewed in a separate commit.
I would skip the change, because it takes more time to fix than just re-write the README.md that will happen later anyway.